setup.py

```diff
@@ -3,7 +3,8 @@
 processing them.
 """
 import os
-from setuptools import setup, find_packages
+
+from setuptools import find_packages, setup
```

Comment:
Can we exclude `setup.py` from `coverage.py`?

Reply:
Sure thing! I tweaked the `.coveragerc` slightly (removed some outdated paths and told it our source lives under `rq/`), then passed that wherever we are calling `pytest` with `--cov`.